static/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -118,14 +118,19 @@ class EventMetas extends Component<Props, State> {
       />
     );
 
-    const httpStatus = <HttpStatus event={event} />;
-
     return (
       <Projects orgId={organization.slug} slugs={[projectId]}>
         {({projects}) => {
           const project = projects.find(p => p.slug === projectId);
+          const isBackendProject =
+            !!project?.platform && backend.includes(project.platform as any);
```

Comment:
What about mobile transactions?

Reply:
Also there's ways to avoid the `any` cast that I've seen for `platform` but it's annoying so I think this is an acceptable place to use it

Reply:
Not including mobile transactions, since HTTP status is a backend-only property! For `any`, an inline `as any` here is for sure the lesser of many evils

Reply:
I believe that `transaction.status` is its own property, separate from HTTP status:

https://develop.sentry.dev/sdk/event-payloads/properties/status/

To be fair though, I'm unsure if these statuses are actually being leveraged by the mobile SDKs. We should still be wary of the possibility that hiding this from mobile users as well may cause confusion for them (if they have statuses other than `ok`, which is what I've been seeing in our [own app ](https://sentry.sentry.io/discover/android-symbol-collector-mobile:ad2b87ba2eda47e994d848efd318a0f6/?field=title&field=event.type&field=project&field=user.display&field=timestamp&field=replayId&field=transaction.status&homepage=true&name=All+Events&project=5953206&query=&sort=-timestamp&statsPeriod=14d&yAxis=count%28%29))

Reply:
Noted!